models/comments.model.js

```diff
@@ -16,3 +17,21 @@ exports.selectCommentsByArticleId = (article_id) => {
       }
     });
 };
+
+exports.setComment = (article_id, username, body) => {
+  if (!username || !body) {
+    return Promise.reject({
+      status: 400,
+      msg: "Invalid data provided for comment",
+    });
```

Comment:
Throwing a custom error is OK here but actually it can dealt with by psql. It's going to be 23502 error